tests/playwright-test/loader.spec.ts

```diff
@@ -238,7 +238,7 @@ test('should fail to load ts from esm when package.json has type module', async
   });
 
   expect(result.exitCode).toBe(1);
-  expect(result.output).toContain('Cannot import a typescript file from an esmodule');
+  expect(result.output).toContain('Unknown file extension ".ts"');
```

Comment:
It would be ideal to point to our docs here, wdyt?

Reply:
I checked that searching exact error brings me to a good stack overflow page.